cmd/test/main.go

```diff
@@ -169,7 +202,6 @@ func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error
 	if len(invalidK8sFiles) > 0 {
 		err = ctx.Evaluator.UpdateFailedK8sValidation(invalidK8sFiles, createEvaluationResponse.EvaluationId, stopEvaluation)
 		if err != nil {
-			fmt.Println(err.Error())
 			return err
```

Comment:
you removed the err print but didn't defer it in the current scope. what will happen with errors in this case?